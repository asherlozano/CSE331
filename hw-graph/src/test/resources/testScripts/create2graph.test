# Create 2graph test file
# Test that 2 graphs can be created
# A graph can be connected to nodes and labeled edges


# Create a graph
CreateGraph g1
CreateGraph g2

# Print the graphs nodes
ListNodes g1
ListNodes g2