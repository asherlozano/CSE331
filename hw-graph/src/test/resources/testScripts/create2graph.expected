# Create 2graphs expected file
# Test that 2 graphs can be created
# A graph can be connected to nodes and labeled edges

# Create a graph
created graph g1
created graph g2

# Print the nodes in the graph
g1 contains:
g2 contains: